internal/pkg/config/config.go

```diff
@@ -32,56 +32,56 @@ type Config struct {
+	WARCPoolSize                 int      `mapstructure:"warc-pool-size"`
+	WARCDedupeSize               int      `mapstructure:"warc-dedupe-size"`
+	CDXDedupeServer              string   `mapstructure:"warc-cdx-dedupe-server"`
+	CDXCookie                    string   `mapstructure:"warc-cdx-cookie"`
+	HQAddress                    string   `mapstructure:"hq-address"`
+	HQKey                        string   `mapstructure:"hq-key"`
+	HQSecret                     string   `mapstructure:"hq-secret"`
+	HQProject                    string   `mapstructure:"hq-project"`
+	HQStrategy                   string   `mapstructure:"hq-strategy"`
+	HQBatchSize                  int      `mapstructure:"hq-batch-size"`
+	HQBatchConcurrency           int      `mapstructure:"hq-batch-concurrency"`
+	DisableHTMLTag               []string `mapstructure:"disable-html-tag"`
+	ExcludeHosts                 []string `mapstructure:"exclude-host"`
+	IncludeHosts                 []string `mapstructure:"include-host"`
+	IncludeString                []string `mapstructure:"include-string"`
+	ExcludeString                []string `mapstructure:"exclude-string"`
+	ExclusionFile                []string `mapstructure:"exclusion-file"`
+	WorkersCount                 int      `mapstructure:"workers"`
+	MaxConcurrentAssetsPerWorker int      `mapstructure:"max-concurrent-assets-per-worker"`
+	MaxHops                      int      `mapstructure:"max-hops"`
+	MaxRedirect                  int      `mapstructure:"max-redirect"`
+	MaxRetry                     int      `mapstructure:"max-retry"`
+	HTTPTimeout                  int      `mapstructure:"http-timeout"`
+	HTTPReadDeadline             int      `mapstructure:"http-read-deadline"`
+	CrawlTimeLimit               int      `mapstructure:"crawl-time-limit"`
+	CrawlMaxTimeLimit            int      `mapstructure:"crawl-max-time-limit"`
+	MinSpaceRequired             int      `mapstructure:"min-space-required"`
+	KeepCookies                  bool     `mapstructure:"keep-cookies"`
+	Headless                     bool     `mapstructure:"headless"`
+	JSON                         bool     `mapstructure:"json"`
+	API                          bool     `mapstructure:"api"`
+	Prometheus                   bool     `mapstructure:"prometheus"`
+	DomainsCrawl                 []string `mapstructure:"domains-crawl"`
+	CaptureAlternatePages        bool     `mapstructure:"capture-alternate-pages"`
+	DisableLocalDedupe           bool     `mapstructure:"disable-local-dedupe"`
+	CertValidation               bool     `mapstructure:"cert-validation"`
+	DisableAssetsCapture         bool     `mapstructure:"disable-assets-capture"`
+	UseHQ                        bool     // Special field to check if HQ is enabled depending on the command called
+	HQRateLimitingSendBack       bool     `mapstructure:"hq-rate-limiting-send-back"`
+	NoBatchWriteWAL              bool     `mapstructure:"ultrasafe-queue"`
+	Handover                     bool     `mapstructure:"handover"`
```

Comment:
Should be removed

Reply:
Nothing to do with archiver, let's not stuff this PR with unrelated (but overall relevant) stuff